plugins/aggregators/basicstats/basicstats.go

```diff
@@ -146,6 +147,9 @@ func (m *BasicStats) Push(acc telegraf.Accumulator) {
 			if config.mean {
 				fields[k+"_mean"] = v.mean
 			}
+			if config.sum {
+				fields[k+"_sum"] = (v.mean * v.count)
```

Comment:
This seemed the simplest way to implement it.

Reply:
And cheapest for backwards compatibility case